### ISSUE TITLE
Make theme overridable

### ISSUE THREAD

**Issue by kprzerwa** [inveniosoftware/react-invenio-app-ils#46]
One possible solution is to change the Semantic-UI theme in `react-invenio-app-ils` and have it as `packaged` theme.
Since Semantic-UI would normally require to have `package` theme in their `semantic-less` repo, we will have to copy the `config` file from that module and adapt it to our needs.

## Semantic-UI limitation

If you check the `semantic-less` node module, you will find that the`Craco Less` plugin is adding a Webpack alias `"../../theme.config$" = path.join(context.paths.appSrc, "/semantic-ui/theme.config")`. The path to the `theme.config` is hardcoded.
That file contains the following:
```
@themesFolder : 'themes';
@siteFolder  : '../../src/semantic-ui/site';
@import (multiple) "~semantic-ui-less/theme.less";
```
And then the `theme.less` contains:
```
/* Packaged site.variables */
@import "@{themesFolder}/@{site}/globals/site.variables";
```
Basically the packaged theme folder is hardcoded to `node_modules/semantic-ui-less/...`.

## Solution

Copy/paste the `default` theme folder from `node_modules/semantic-ui-less/` and rename the `site` to `ils` so it becomes the packaged theme, to have:
```
semantic-ui/themes/
  default/
  ils/
```
Unfortunately the `default` is required because it is hardcoded. Adjust config files as needed.

In the overlay then, we will be able to create a site theme by changing the `theme.config`:
`@themesFolder : '~node_modules/react-invenio-app-ils/semantic-ui/themes/ils';`
